README.md

```diff
@@ -111,47 +113,40 @@ can see the routes with ```rake routes```;
   DB_TEST_PASSWORD=[Use your local Postgres password, if any, for bfr_webapp_user]
   ```
 
-5. **Create database:**
+
+5. **Load Development Database Schema:**
 
   ```
-  bundle exec rake db:create
+  bundle exec rake db:schema:load
   ```
 
-6. **If you receive an encoding error, go to db console and:**
+  **Warning:** _`bundle exec rake db:migrate` currently does not work due to default scopes an improper order of columns added to the database with migrations. `bundle exec rake db:schema:load` will build your tables instead._
+
+6. **Seed Development Database**
 
   ```
-  SET CLIENT_ENCODING TO 'WIN1252';
+  bundle exec rake db:seed
   ```
+  **Note:** _This creates an admin volunteer and other required bits. You should look it over._
 
-7. **Load Database Schema:**
+7. **Load Test Database Schema**
 
   ```
-  bundle exec rake db:schema:load
+  bundle exec rake db:schema:load RAILS_ENV=test
   ```
 
-  **Warning:** _`bundle exec rake db:migrate` currently does not work due to default scopes an improper order of columns added to the database with migrations._
-
-8. **Seed:**
+8. **Seed Test Database**
 
   ```
-  bundle exec rake db:seed
+  bundle exec rake db:seed RAILS_ENV=test
```

Comment:
I'm not 100% sure you need to do this, but this is a nice improvement, imo. Thanks!